src/core/nsfs.js

```diff
@@ -29,13 +29,16 @@ Help:
 const USAGE = `
 Usage:
 
-    noobaa-core nsfs <root-path> [options...]
+    noobaa-core nsfs <root-path> <fs-path> <protocol> <bucket-name> [options...]
```

Comment:
@romayalon the nsfs command is meant to behave differently - not to export a single bucket, but instead to export a root dir so that every subdir is a bucket. Can you preserve this behavior?

Reply:
I removed these flags and kept nsfs with the minimal changes I can, but the behavior won't be exactly the same since there are more changes in namespace_fs for exporting a single bucket.